### PR TITLE
Do not abbreviate. (and use single quotes where you can)

### PR DESCRIPTION
**Pull request by alboyadjian**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1927,6 +1927,39 @@ no parameters.
   `lib/hello_world/hello_world.rb`.
 <sup>[[link](#snake-case-dirs)]</sup>
 
+* <a name="do-not-abbreviate"></a>
```
Comment (hurtstotouchfire) [inline README.md:1930]: I am definitely on-board in principle here, but I am also on board with 120 line lengths. If we want both short lines and long var names, can we add a section on the appropriate line-breaking style? And maybe refer to that in here.

**Comment by davexunit** [comment]
In general I would agree to avoid abbreviations, but sometimes abbreviations are valid, particularly when speaking about math.  I really don't mind `i` instead of `index`, for example, or `n` to refer to an arbitrary numeric value, or `k` and `v` to refer to a key/value when iterating over a Hash.

**Comment by alboyadjian** [comment]
I get your point.  Part of the goal of having some standards defined and being reasonably strict about enforcing them is to improve the inter-coder consistency of our finished product.  For "index" for example, I've seen people use "i", "n", "index", and "idx".  If we want to define some exceptions to the rule, I'd like to try to delineate them, and then encourage using the abbreviation consistently, vs having a mishmash.  If we just say "it's OK sometimes", everyone is going to think that their particular case is a special snowflake.

**Comment by davexunit** [comment]
Yes, I agree.  My comments above are just nitpicks.  I think this patch is good.  Let's merge it?

**Pull request merged**